Spec.md

```diff
@@ -372,6 +380,30 @@ This field defaults to `false`, including for `spec_version`s less than
 `v1.16`, however CKAN clients prior to `v1.16` would only perform strict
 checking.
 
+##### tags
+
+The `tags` field describes keywords that a user or program may use to
```

Comment:
Please add a **(v1.23)** to indicate that this will only be supported by CKAN clients of version v1.23.0 and higher

Reply:
We always go up by two numbers (14, 16, 18, 20, 22). So shouldn't the next be v1.24?

Reply:
Adding 1.24...